### DISCUSSION TITLE
distinguish types of adjectives

### DISCUSSION THREAD

**Discussion by chaouiy** [explosion/spaCy#3890] [New Features & Project Ideas]
It would be great if we can use spacy to distinguish which semantic type of adjective / adverb it is. This is a recurrent use, because we usually need to extract only descriptive adjectives and results are poluted with other types of adjectives. Here is an example list I use to do this manually :
```
       "quantitative": ["half", "some", "several", "much", "all", "many", "whole", "no", "enough", "little", "any", "sufficient", "none", "most", "few"],
        "demonstrative": ["this", "that", "these", "those"],
        "possessive": ["my", "his", "their", "your", "our", "mine", "his", "hers", "theirs", "yours", "ours"],
        "interrogative": ["which", "what", "whose"],
        "distributive": ["each", "every", "either", "neither", "any"],
        "article": ["a", "an", "the"],
```

**Comment by honnibal** [comment]
Most of those are determiners and predeterminers though. If you use the `token.tag` attribute, does the `JJ` (and `JJR`) tag give you what you want?
